remote-build-coordinator/src/test/java/org/jboss/pnc/remotecoordinator/test/BuildCoordinatorFactory.java

```diff
@@ -63,68 +55,70 @@ public BuildCoordinatorBeans createBuildCoordinator(DatastoreMock datastore) {
 
         BuildTaskRepository taskRepository = new BuildTaskRepositoryMock();
 
-        LocalBuildSchedulerMock localBuildScheduler = new LocalBuildSchedulerMock();
-
-        BuildCoordinator coordinator = new RemoteBuildCoordinator(
-                datastoreAdapter,
-                buildStatusChangedEventNotifier,
-                buildSetStatusChangedEventNotifier,
-                localBuildScheduler,
-                taskRepository,
-                systemConfig,
-                groupBuildMapper,
-                buildMapper);
-        localBuildScheduler.setBuildCoordinator(coordinator);
-
-        SetRecordUpdateJob setJob = new SetRecordUpdateJob(taskRepository, datastore, coordinator);
-
-        return new BuildCoordinatorBeans(taskRepository, coordinator, setJob);
+        // TODO RexBuildSchedulerMockMock localBuildScheduler = new RexBuildSchedulerMockMock();
```

Comment:
Please look at the tests extending the `org.jboss.pnc.remotecoordinator.test.ProjectBuilder`. If we still need them, they should be moved to the integration-tests. I would drop them as they are testing the build coordination which is now happening in the Rex and the scenarios should be tested in the Rex.

Reply:
I'd rather move them to integration tests.

Reply:
Let's wait untill you finish the result storing to see where is the best place for these. Without modifications, they don't fit well in the integration tests, because they are using the coordinator directly, rather than the endpoints.